include/constants/battle.h

```diff
@@ -101,6 +101,8 @@
 #define STRUGGLING_MOVE_4           (1 << 3)
 #define STRUGGLING_ALL              (STRUGGLING_MOVE_1 | STRUGGLING_MOVE_2 | STRUGGLING_MOVE_3 | STRUGGLING_MOVE_4)
 
+#define SYSTEM_CONTROL_SKIP_DAMAGE_SPRITE_BLINK (1 << 6)
```

Comment:
I expect that this will necessitate its own file at some point, so this name is subject to change.